packages/re6stnet.rb

```diff
@@ -0,0 +1,39 @@
+require 'package'
+class Re6stnet < Package
+  description 'Resilient, Scalable, IPv6 Network'
+  homepage 'https://re6st.nexedi.com/'
+  version '0.501'
+  source_url 'https://lab.nexedi.com/nexedi/chromebrew-buildout-re6st/repository/archive?sha=master'
```

Comment:
Since this repository doesn't have any tagged released and since the only branch is master, it would be better to use a hashed commit to lock down the "version".
```
https://lab.nexedi.com/nexedi/chromebrew-buildout-re6st/repository/archive.tar.bz2?ref=ec195cec46f7ed4893bbc8462266d28cec886dad
```
This will grab the latest commit from the master branch.